generators/python/src/fern_python/cli/publisher.py

```diff
@@ -13,12 +13,20 @@ class Publisher:
     def __init__(
         self,
         *,
+        should_format: bool,
         generator_exec_wrapper: GeneratorExecWrapper,
         generator_config: GeneratorConfig,
     ):
+        self._should_format = should_format
         self._generator_exec_wrapper = generator_exec_wrapper
         self._generator_config = generator_config
 
+    def run_ruff_format(self) -> None:
+        self._run_command(
```

Comment:
Does this always get called after `poetry install` if not you might want to call that before to make sure ruff is there to be run